main.go

```diff
@@ -63,7 +67,12 @@ func runE(cmd *cobra.Command, args []string) error {
 		ctx.String(http.StatusOK, "Ok")
 	})
 
-	router.POST("/webhooks", rpc.Webhooks)
+	router.POST("/webhooks", func(c *gin.Context) {
+		// get the '--excluderepos' flag value
+		excludeRepos, _ := cmd.Flags().GetStringArray("excluderepos")
+
+		rpc.Webhooks(c, excludeRepos)
+	})
```

Comment:
This should go back to being:
```suggestion
	router.POST("/webhooks", rpc.Webhooks)
```
and instead you pass in `cfg.ExcludeRepos` into the `rpc.New` function and add it to that struct